boards/chronos/include/periph_conf.h

```diff
@@ -26,9 +26,16 @@ extern "C" {
 #endif
 
 /**
- * @brief   Clock configuration
+* @todo
+*
+* @{
+* Move all clock configuration code here from the board.h
+* @}
+*/
+/**
+ * @name   Clock configuration
  *
- * @todo    Move all clock configuration code here from the board.h
```

Comment:
I tried several combinations, including the ones you propose, but none worked. That's why I moved the todo outside the name group